baksmali/src/main/java/org/jf/baksmali/Adaptors/ClassDefinition.java

```diff
@@ -64,7 +64,7 @@ public boolean hadValidationErrors() {
 
     @Nonnull
     private HashSet<String> findFieldsSetInStaticConstructor() {
-        HashSet<String> fieldsSetInStaticConstructor = new HashSet<String>();
```

Comment:
A better solution here is to make findFieldSetInStaticConstructor static, and pass in classDef. This avoids the name ambiguity, since a static method can't access an instance field.